### PR TITLE
[code] fix: add oban logger + rename configs

### PR DESCRIPTION
**Pull request by grejdi-mbta**
This PR adds logging for Oban through [telemetry](https://github.com/beam-telemetry/telemetry). It also renames the glue database variables to reflect the new name `glue_database_incoming`.

Lastly, one minor fix to remove some unnecessary querying.

### CODE REVIEW

File: ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/oban_logger.ex [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  @spec handle_event(
+          :telemetry.event_name(),
+          :telemetry.event_measurements(),
+          :telemetry.event_metadata(),
+          :telemetry.handler_config()
+        ) :: any()
+  def handle_event([:oban, :job, :start], _measure, meta, _config) do
+    Logger.info(
+      "#{@log_prefix} [#{meta.queue}] Start: args=#{inspect(meta.args, charlists: :as_lists)}"
```
Comment (grejdi-mbta) [inline ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/oban_logger.ex:18]: Needed `charlists: :as_lists` in order to print out lists of integers correctly.

File: ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_table.ex [hunk 2]
```diff
@@ -52,28 +52,33 @@ defmodule ExCubicOdsIngestion.Schema.CubicOdsTable do
...
-  @spec filter_to_existing_prefixes(Enumerable.t()) :: [{String.t(), t()}]
+  @spec filter_to_existing_prefixes(Enumerable.t()) :: [{String.t(), t()}] | []
   def filter_to_existing_prefixes(prefixes) do
-    # strip any change tracking suffix
-    without_change_tracking =
-      prefixes
-      |> MapSet.new(&String.replace_suffix(&1, "__ct/", "/"))
-      |> Enum.to_list()
+    # in order to prevent querying with an empty list, we just return an empty list
+    if Enum.empty?(prefixes) do
```
Comment (grejdi-mbta) [inline ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_table.ex:58]: This wasn't causing an issue, but it was making an extra query, which was annoying.

File: ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_table.ex [hunk 3]
```diff
@@ -52,28 +52,33 @@ defmodule ExCubicOdsIngestion.Schema.CubicOdsTable do
   @doc """
   Given an enumerable of S3 prefixes, return those prefixes which represent a #{__MODULE__} and their table.
   """
-  @spec filter_to_existing_prefixes(Enumerable.t()) :: [{String.t(), t()}]
+  @spec filter_to_existing_prefixes(Enumerable.t()) :: [{String.t(), t()}] | []
```
Comment (paulswartz) [inline ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_table.ex:55]: nitpick(non-blocking): `[]` was already included in the type: `[X]` is the type of lists of X with 0 or more items. If we'd wanted to say that there is always at least one X, that's `[X, ...]`.

File: ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/workers/ingest.ex [hunk 4]
```diff
@@ -11,7 +11,7 @@ defmodule ExCubicOdsIngestion.Workers.Ingest do
 
   require Logger
 
-  @log_prefix "[ex_cubic_ods_ingestion][workers][ingest]"
+  @log_prefix "[ex_cubic_ods_ingestion] [workers] [ingest]"
```
Comment (paulswartz) [inline ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/workers/ingest.ex:14]: question: I don't see this used anywhere?

Comment (paulswartz) [reply ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/workers/ingest.ex:14]: nevermind, I missed what the edit was here.

**Review by paulswartz — APPROVED** [review]
🍰

File: ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_table.ex (hunk 3 above)
Comment (grejdi-mbta) [reply ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_table.ex:55]: I'll remove it.

**Pull request merged**